charts/tempo-distributed/values.yaml

```diff
@@ -1006,7 +1006,7 @@ queryFrontend:
     tag: null
   service:
     # -- Port of the query-frontend service
-    port: 16686
+    port: 3200
```

Comment:
Should it not be 7777?

Reply:
okay, so getting back to this, the `service.port` properties are nested under `queryFrontend`, which has the static `3200` port for (i guess) just metrics via HTTP. The property's description also refers to *query-frontend*, not *tempo-query*, so this whole field seems a bit misplaced. I don't know if it's actually needed for something, but if it should just rectify the broken old port to the new (gRPC) port of tempo-query, we can set it to `7777` in the values and use that value in the template, sure.

However I think the confusion I had here is really indicative of drift of the upstream `tempo-query` service and the helm chart's use of it.

For now, I'll set this to 7777 and carry that to the service template.